hedera-node/hedera-app/src/main/java/com/hedera/node/app/blocks/impl/PairedStreamBuilder.java

```diff
@@ -160,6 +160,11 @@ public int getNumAutoAssociations() {
         return blockStreamBuilder.getNumAutoAssociations();
     }
 
+    @Override
+    public ScheduleID scheduleID() {
+        return recordStreamBuilder.scheduleID();
```

Comment:
Prefer delegation to `blockStreamBuilder` here (although it's not _terribly_ important since `streamMode=BOTH` is temporary).